api/server/form.go

```diff
@@ -43,7 +44,7 @@ func archiveFormValues(r *http.Request, vars map[string]string) (archiveOptions,
 	}
 
 	name := vars["name"]
-	path := r.Form.Get("path")
+	path := filepath.FromSlash(r.Form.Get("path"))
```

Comment:
THANK YOU! I completely forgot to do this (even though I mentioned it in the client-side code).